packages/beacon-node/test/sim/withdrawal-interop.test.ts

```diff
@@ -350,3 +349,26 @@ describe("executionEngine / ExecutionEngineHttp", function () {
     console.log("\n\nDone\n\n");
   }
 });
+
+async function retrieveCanonicalWithdrawals(bn: BeaconNode, fromSlot: Slot, toSlot: Slot): Promise<number> {
```

Comment:
@g11tech Can you review if this assertion is good for your test? Re-wrote the previous event based, into an after the fact one

Reply:
yes looks good